x-pack/package.json

```diff
@@ -27,6 +27,7 @@
     "**/@types/node": "10.12.27"
   },
   "devDependencies": {
+    "@hapi/basic": "^5.1.1",
```

Comment:
Removed, no longer an issue.
But in general though, if all the tests pass and this is only used in "devDependencies" and enabled in an integration test... do we not feel same enough to add such things?

In this case it was avoidable, but in the future it might not be.

Just trying to gauge how confident we are in our testing process so I know where to focus my attention.